### PR TITLE
os/arch/arm/src/amebad: Fix the 'mixed uart log' issue

### PR DESCRIPTION
**Pull request by AdamZhang0124**
This PR fixes the 'mixed uart log' issue which happens when multiple functions use log_uart at the same time.

### CODE REVIEW

**Comment by AdamZhang0124** [comment]
This PR is linked with https://github.com/Samsung/TizenRT/pull/5061

**Comment by AdamZhang0124** [comment]
Hi @sunghan-chang @jeongchanKim @jeongarmy ,
Please refer to this PR, all the comment questions mentioned in #5061 have been resolved.
Thanks!

**Review by jeongchanKim — APPROVED** [review]
LGTM

File: os/arch/arm/src/amebad/amebad_serial.c [hunk 1]
```diff
@@ -114,7 +114,7 @@
 #elif defined(CONFIG_UART1_SERIAL_CONSOLE)
 #define CONSOLE_DEV             g_uart1port             /* UART1 is console */
 #define TTYS0_DEV               g_uart1port             /* UART1 is ttyS0 */
-#define CONSOLE                 UART1_DEV
+#define CONSOLE                 UART3_DEV
```
Comment (sunghan-chang) [inline os/arch/arm/src/amebad/amebad_serial.c:117]: In the comments of amebad_serial.c, UART3_DEV is KM0 luart.
```
/*
 * Available UARTs in AmebaD
 * UART0_DEV: KM4 uart0
 * UART1_DEV: KM4 uart1_bt
 * UART2_DEV: KM0 log uart
 * UART3_DEV: KM0 luart
 */
```
UART1 is connected to KM0? not KM4?

Comment (AdamZhang0124) [reply os/arch/arm/src/amebad/amebad_serial.c:117]: Yes, the UART1 actually associates with the KM0 luart.
If necessary, I will submit another PR to completely change all the UART1 notation to UART3 instead.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**